lib/deep-equal.js

```diff
@@ -152,6 +153,23 @@ function deepEqualCyclic(actual, expectation, match) {
             return isSubset(actualObj, expectationObj, deepEqual);
         }
 
+        if (isMap(actualObj) || isMap(expectationObj)) {
+            if (
+                !isMap(actualObj) ||
+                !isMap(expectationObj) ||
+                actualObj.size !== expectationObj.size
+            ) {
+                return false;
+            }
+
+            return Array.from(actualObj.keys()).every(function(key) {
```

Comment:
I just checked: Apparently `Array.from` isn’t supported by our favorite browser.

Reply:
The `Map` check that I suggested should filter that out also.

Reply:
You're right. It is from ES2015.

I guess there are a few options:

* accept that using ES2015 globals inside something guarded by a test for a different ES6 global
* add a polyfill for `Array.from`
* write the code in a way that uses ES5.1

I don't particularly want to add yet another polyfill. I am most inclined to write the code in strict ES5.1, even if it is working on `Map` from ES2015.

Reply:
I agree.

Reply:
Pushed a commit to do this